### DISCUSSION TITLE
Cannot read properties of undefined (reading 'getViewLineContent')

### DISCUSSION THREAD

**Discussion by zachstultz** [community/community#81375] [Copilot Conversations]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/8385256/4ce9202c-7609-4da4-9d87-c20a0082c59e)

**Comment by muleyprasad** [comment]
How did you resolve it? is there any workaround for this?

**Comment by jonas-martinez** [comment]
I'm also experiencing this issue when asking it to convert a js file to a java file (170 lines of code).

**GitHub Copilot version** : v1.143.0
**GitHub Copilot Chat version** : v0.11.1

**Comment by frederickk** [comment]
Also having the same error, and haven't found a work around.

- MacOS VSCode `Version: 1.85.1 (Universal)`
- GitHub Copilot `v1.145.0`
- GitHub Copilot Chat `v0.11.1`

**Comment by thecontstruct** [comment]
Just ran into this as well and got around it by adding about 5-6 whitespace lines to the bottom of the file, selecting only the code i want it to modify, then asked copilot to convert it from JS to TS. 

I was only dealing with 29 lines of code so the milage may vary on this approach.
